Makefile

```diff
@@ -48,6 +49,8 @@ build: prebuild
 	@$(MAKE) build/deb/$(NAME)_$(VERSION)_amd64.deb
 	@$(MAKE) build/deb/$(NAME)_$(VERSION)_arm64.deb
 
+build-native: build/$(SYSTEM_NAME)/$(NAME)-$(HARDWARE_GENERIC)
+
```

Comment:
thought: We can omit this change, I just found it very handy to be able to run make build-native on both my mac and Linux machines 🙂

Reply:
Please remove the makefile changes.

Reply:
Should I submit this as a separate PR or not at all?

Reply:
You can PR it separately.